### PR TITLE
Update UPN value format

### PR DESCRIPTION
**Pull request by VLG17**
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/8548

### CODE REVIEW

**Comment by illfated** [comment]
(ref. #8548 )

File: windows/security/identity-protection/hello-for-business/hello-hybrid-aadj-sso-cert.md [hunk 1]
```diff
@@ -656,7 +656,7 @@ Sign-in a workstation with access equivalent to a _domain user_.
 11. Select **Enroll to Windows Hello for Business, otherwise fail (Windows 10 and later)** from the **Key storage provider (KSP)** list.
 12. Select **Custom** from the **Subject name format** list.
 13. Next to **Custom**, type **CN={{OnPrem_Distinguished_Name}}** to make the on-premises distinguished name the subject of the issued certificate.
-14. Specify **User Principal Name (UPN)** as a **Subject Alternative Name** value.
+14. Specify **User Principal Name (UPN)** as a **Subject Alternative Name** value. Use **{{UserPrincipalName}}** format for **User Principal Name (UPN)** value.
```
Comment (illfated) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-aadj-sso-cert.md:659]: cosmetic/readability:
```suggestion
14. Specify **User Principal Name (UPN)** as a **Subject Alternative Name** value. Use the **{{UserPrincipalName}}** format for the **User Principal Name (UPN)** value.
```

**Comment by illfated** [comment]
@VLG17 : I was just scrolling up in the PR document content to read the metadata section (yes, I could have opened the file in a new tab instead), but it appears that GitHub interprets MarkDown code block fence abuse in a slightly odd way: if the 3 backticks are placed at the beginning and end of a line, instead of on a blank line before and after, the source view text turns blue as if it is a special form of scripting.

Would it be too time-consuming to look into that detail and consider adapting the code block fences, so it would have to be done in a separate PR? (Yes, I appreciate that it is quite outside the scope of this PR, but I thought I would ask, just to be on the safe side.)

**Review by illfated — APPROVED** [review]
Looks good to me so far, just waiting for final review by the author.

**Comment by VLG17** [comment]
> @VLG17 : I was just scrolling up in the PR document content to read the metadata section (yes, I could have opened the file in a new tab instead), but it appears that GitHub interprets MarkDown code block fence abuse in a slightly odd way: if the 3 backticks are placed at the beginning and end of a line, instead of on a blank line before and after, the source view text turns blue as if it is a special form of scripting.
> 
> Would it be too time-consuming to look into that detail and consider adapting the code block fences, so it would have to be done in a separate PR? (Yes, I appreciate that it is quite outside the scope of this PR, but I thought I would ask, just to be on the safe side.)

@illfated To be honesc, i don't really know/understand what i should change and where. I looked at the changes and don't see what you're seeing, but then again maybe i'm not looking in the same place. Could you share a printscreen of what you're seeing and where you're seeing this exactly, so i can replicate it and understand better?

**Comment by illfated** [comment]
Very well, maybe it is only in my own browser and not for anybody else. I admit that I am using a GitHub-specific addon, Enhanced GitHub (https://chrome.google.com/webstore/detail/enhanced-github/anlikcnbgdeidpacdbdljnabclhahhmd).

Here is the first appearance of the change in my view of the top of the file (from line 288, the blue text starts after the odd use of 3 backticks on line 293:

image

The visual issue (at least for me) continues all the way through the entire page of the source view, never cancelled, only continued by repeated use of 3 backticks where 1 pair (start + end) used to encapsulate stand-alone code lines would have sufficed.

From my experience, it should either be 1 MarkDown backtick pair used as encapsulation on the same line as the encapsulated code, or 3 backticks on a blank line before (with or without syntax highlighting) and 3 backticks on the first blank line _after_ the code to end the encapsulation.

I just thought I should ask if this is something that interests you, or if I should rather do something about it on my own. This is mostly about MarkDown Codestyle anyway, so feel free to call it "out of scope" if you do have more than enough to deal with already. 🙂

**Comment by VLG17** [comment]
> Very well, maybe it is only in my own browser and not for anybody else. I admit that I am using a GitHub-specific addon, Enhanced GitHub (https://chrome.google.com/webstore/detail/enhanced-github/anlikcnbgdeidpacdbdljnabclhahhmd).
> 
> Here is the first appearance of the change in my view of the top of the file (from line 288, the blue text starts after the odd use of 3 backticks on line 293:
> 
> image
> 
> The visual issue (at least for me) continues all the way through the entire page of the source view, never cancelled, only continued by repeated use of 3 backticks where 1 pair (start + end) is used to encapsulate stand-alone code lines, where a single pair would have sufficed.
> 
> From my experience, it should either be 1 MarkDown backtick pair used as encapsulation on the same line as the encapsulated code, or 3 backticks on a blank line before (with or without syntax highlighting) and 3 backticks on the first blank line _after_ the code to end the encapsulation.
> 
> I just thought I should ask if this is something that interests you, or if I should rather do something about it on my own. This is mostly about MarkDown Codestyle anyway, so feel free to call it "out of scope" if you do have more than enough to deal with already. 🙂

It's clear now what you mean, but i still cannot reproduce it on my side. I would appreciate if you could do the changes needed, because it's something you understand better then me.

One thing worth mentioning, the 3 backticks did not come from me (unless accidentally).

**Comment by illfated** [comment]
Sure thing. I presumed that the less favorable MarkDown formatting was not done by you. I may be overzealous a lot of the time, thinking "While we are at it, maybe we can add some more improvements in the PR before it gets merged.", but never mind, I am often overthinking things, especially when it comes to codestyle & formatting.

Very well, I see that this is something I can handle on my own, just waiting to see your changes merged before I start editing the source file.

**Review by illfated — APPROVED** [review]

**Comment by illfated** [comment]
#sign-off

**Comment by nam31** [comment]
@mapalko PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by illfated** [comment]
@nam31 : Please note: Even though mapalko is the current author of many MS Docs pages, he usually does not have merging rights. That being said, we are always glad to have him do a final review before handing it over to the document manager.

**Comment by VLG17** [comment]
@illfated Would you be able to help with the conflicts that need to be resolved?

**Comment by illfated** [comment]
Thank you for asking. Maybe. Let me see: You started this PR on November 6, let us see which changes were made.
- https://github.com/MicrosoftDocs/windows-itpro-docs/blob/public/windows/security/identity-protection/hello-for-business/hello-hybrid-aadj-sso-cert.md
- https://github.com/MicrosoftDocs/windows-itpro-docs/commits/public/windows/security/identity-protection/hello-for-business/hello-hybrid-aadj-sso-cert.md

That looks like mainly 1 PR change (including 3 extra commits within that PR):
- **updated steps for Intune** #8732
- final commit after final review: https://github.com/MicrosoftDocs/windows-itpro-docs/commit/13294d73d61a328ace8df07518939a88077b87d2

**Comment by illfated** [comment]
Oh well, might just be easier to add your changes to the current file. 
Going to make a PR to update yours, then add back your changes in this PR.

**Comment by illfated** [comment]
Hmm. On second thought, it looks like the PR #8732 has already added the necessary change to that line, as well as modifying the preceding text:
```PowerShell
12. Specify **User Principal Name (UPN)** as a **Subject Alternative Name** parameter. Set its value as {{UserPrincipalName}}.
```
https://github.com/MicrosoftDocs/windows-itpro-docs/blob/5d460c78b68d3ffe04baeb29f672134f584754e4/windows/security/identity-protection/hello-for-business/hello-hybrid-aadj-sso-cert.md#L653-L655

Could you verify/confirm if the change is already published, making this PR redundant?
- https://github.com/MicrosoftDocs/windows-itpro-docs/blob/public/windows/security/identity-protection/hello-for-business/hello-hybrid-aadj-sso-cert.md#create-a-scep-certificate-profile

**Comment by VLG17** [comment]
@illfated You are right, the changes are reflected in the article. It looks good, thus making the PR redundant, as you also mentioned. Should i just close it?

**Comment by illfated** [comment]
Since you seem to agree, I think that would be the right thing to do. Very well. Better luck next time. 🙂 👍🏻

**Pull request closed without merging**